hm10_2.py

```diff
@@ -0,0 +1,32 @@
+""" Task 2: return number """
+
+
+def decorator(_any):
+
+    """ Function takes arguments _r """
+
+    def wrapper(*args):
+
+        """ Function takes arguments in any value """
+
+        lst = []
+        for x in args:
+            lst.append(x)
+        for x in lst:
+            if not isinstance(x, (int, float)):
+                print("Result is not numeric")
+                break
+        return False
+    return wrapper
+
+
+@decorator
+def my_func(*args):
+
+    """ Function shows sum of arguments """
+
+    a = sum(args)
+    return a
+
+
+my_func(1, 9.1, "k")
```

Comment:
`assert ...`

Reply:
Used args, ok
I don’t understand how to use assert correctly, as it duplicates information. Please, help with the correct use with assert without duplicates information